File: examples/fizzbuzz/fizzbuzz.cpp
#include <iostream>
#include <sstream>

using namespace std;

int main() {

    string endS;

    cout << "Enter end of fizzbuzz: ";
    cin >> endS;

    int end;

    stringstream convert(endS);

    convert >> end;

    for(int i=1; i<=end; i++) {
        if (i%3==0 && i%5 == 0) {
            cout << "Fizzbuzz" << endl;
        } else if (i%3==0) {
            cout << "Fizz" << endl;
        } else if (i%5==0) {
            cout << "Buzz" << endl;
        } else cout << i << endl;
    }

    return 0;
}

File: examples/fizzbuzz/fizzbuzz.py
from PyCpp import *

def main() -> int:

    end = cin >> "Enter end of fizzbuzz: "

    try:
        end = stringstream(end) >> 1 # convert 'end' to int
    except Exception as e:
        raise e

    for i in range(1, end+1):
        if i%3 == 0 and i%5 == 0:
            cout << "Fizzbuzz" << endl
        elif i%3 == 0:
            cout << "Fizz" << endl
        elif i%5 == 0:
            cout << "Buzz" << endl
        else:
            cout << i << endl

    return 0;

main()